2022-01-25 11:20:34.612 | DEBUG    | httprunner.client:log_print:40 - 
================== request details ==================
method   : POST
url      : https://tester1.zhixueyun.com/api/v1/exam/exam-record/add-user-to-exam
headers  : {
    "User-Agent": "python-requests/2.25.1",
    "Accept-Encoding": "gzip, deflate",
    "Accept": "*/*",
    "Connection": "keep-alive",
    "Authorization": "Bearer__c0909a02d4a8aa71f02b2f2c302f1894",
    "content-type": "application/x-www-form-urlencoded; charset=UTF-8",
    "uri": "exam/exam",
    "Content-Length": "84"
}
cookies  : {}
body     : ids=5f0870a7-d55d-4a31-8782-1ac8d9690efc&examId=d2b32a87-0aeb-4319-a48a-520e9739d508

2022-01-25 11:20:34.612 | DEBUG    | httprunner.client:log_print:40 - 
================== response details ==================
status_code : 200
headers  : {
    "Date": "Tue, 25 Jan 2022 03:20:35 GMT",
    "Content-Type": "application/json;charset=UTF-8",
    "Content-Length": "21",
    "Connection": "keep-alive",
    "Strict-Transport-Security": "max-age=15724800; includeSubDomains",
    "Access-Control-Allow-Origin": "*",
    "Access-Control-Allow-Credentials": "true"
}
cookies  : {}
encoding : UTF-8
content_type : application/json;charset=UTF-8
body     : {
    "success": "success"
}

2022-01-25 11:20:34,613 | D:\PyCharm\project\httprunner3\common\validate.py[line:76] | INFO: 响应断言:  预期值: 'success' 实际值: 'success', 断言成功!
